src/Pages/Settings.tsx

```diff
@@ -1,10 +1,380 @@
-import React from "react";
+import {
+    Button,
+    Dialog,
+    DialogContent,
+    DialogTitle,
+    TextField,
+    IconButton,
+    Input, 
+    Chip, 
+    Autocomplete, 
+  
+  } from "@mui/material";
+  import * as React from "react";
+  import { useFormik } from "formik";
+  import * as yup from "yup";
+  import awsconfig from '../aws-exports'
+  import {Amplify} from 'aws-amplify'
+  import { createUser } from "../backend/mutations/userMutations";
+  import { CreateUsersModelInput } from "../API";
+  import { CreateUsersPayload } from "../backend/types";
+  Amplify.configure(awsconfig)
+
+  
+
+  ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+  ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+  // temporary lists for choices to user 
+  const select_lang = [
+    { title: 'C++'},
+    { title: 'Java'},
+    { title: 'Python' },
+  ];
+  
+  
+  const select_frame = [
+    { title: 'React'},
+    { title: 'Linux'},
+    { title: 'Shell' },
+  ];
+  
+  const select_category = [
+    { title: 'Cloud'},
+    { title: 'Web Programming'},
+    { title: 'DevOps' },
+  ];
+  ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+  ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+  
+  const frontload: CreateUsersModelInput = {
+    user_name: '',
+    email: '',
+    first_name: '',
+    last_name: '',
+    password: ''
+  }
+  const finalload: CreateUsersPayload = {
+    input: frontload
+  }
```

Comment:
Why is create user functionality in settings

Reply:
if it's change password/ etc. we still need to write specific mutations on the backend to modify vs create

Reply:
The code block is just to reference later for connecting the settings page to the backend, it can be ignored for now